ha-addon/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3.9-slim
```

Comment:
Why python:3.9-slim? I think it's too low.

Reply:
OK, we can use the same than the Dockerfile you used for creating the docker releases
```suggestion
# Use an official Python runtime as a parent image
```